python/translit/cases.py

```diff
@@ -0,0 +1,24 @@
+"""
+Class mimic to enum to keep consistent definition
+all possible transliteration case options.
+"""
+
+
+class Case:
```

Comment:
We should discuss with Vincuk what's the best name for this enum. Is it  alphabet or orthography or something else. No need to do anything here right now.

Reply:
Let's maybe create another issue for that?